### PR TITLE
Cleanup and bug fixes

### PR DESCRIPTION
**Pull request by microfauna**
Fixes several homepage bugs but also cleans up the visuals of BusinessCard view

image

### CODE REVIEW

File: src/components/BusinessCard.vue [hunk 1]
```diff
@@ -33,23 +32,31 @@
...
         <div v-else>{{business.short_description}}</div>
       </div>
-      <div style="flex: 1">
+      <div class="flex flex-row items-end">
+        <div class="text-sm flex flex-row align-baseline flex-grow">
+          <span
+            v-for="tag in tags"
+            :key="tag.id"
+            class="box-border whitespace-no-wrap text-gray-500 pr-2 flex-end"
+            :class="`tag-${tag.name}`"
```
Comment (arecvlohe) [inline src/components/BusinessCard.vue]: Funny how these can be separate but I think you could also just combine them

File: src/components/business/Map.vue [hunk 2]
```diff
@@ -33,7 +33,6 @@ export default {
   components: { MglMap, MglMarker },
   methods: {
     updateCoordinates (e) {
```
Comment (arecvlohe) [inline src/components/business/Map.vue:35]: Kept getting coordinate errors in the console, do I need a key?

**Review by arecvlohe — DISMISSED** [review]

File: src/components/business/Map.vue (hunk 2 above)
Comment (microfauna) [reply src/components/business/Map.vue:35]: i think you do! I can put mine in secrets storage for you. End goal would be google-less but, y'know.

File: src/components/BusinessCard.vue (hunk 1 above)
Comment (microfauna) [reply src/components/BusinessCard.vue]: Combined!

**Pull request merged**